### PR TITLE
fragmentInfo has text instead string for using display mode

### PR DESCRIPTION
**Pull request by yCobanoglu**

### CODE REVIEW

**Comment by yCobanoglu** [comment]
FragmentInfo used to have `matchingLines` this was used for lines which match the transliteration query (in Fragmentarium Frontend) they were string but have to be proper Text to be displayed in Display Mode (similar to editor in Fragmentarium Frontend)

File: ebl/tests/fragmentarium/test_fragment.py [hunk 1]
```diff
@@ -278,30 +278,27 @@ def test_set_text():
...
 GET_MATCHING_LINES_DATA = [
-    ([["KU", "NU"]], [["1'. [...-ku]-nu-ši [...]"]]),
-    ([["U", "BA", "MA"]], [["3'. [... k]i-du u ba-ma-t[a ...]"]]),
-    ([["GI₆"]], [["2'. [...] GI₆ ana GI₆ u₄-m[a ...]"]]),
+    ([["KU", "NU"]], "1'. [...-ku]-nu-ši [...]"),
+    ([["U", "BA", "MA"]], "3'. [... k]i-du u ba-ma-t[a ...]"),
+    ([["GI₆"]], "2'. [...] GI₆ ana GI₆ u₄-m[a ...]"),
     (
         [["GI₆", "DIŠ"], ["U", "BA", "MA"]],
-        [["2'. [...] GI₆ ana GI₆ u₄-m[a ...]", "3'. [... k]i-du u ba-ma-t[a ...]"]],
```
Comment (yCobanoglu) [inline ebl/tests/fragmentarium/test_fragment.py:286]: consecutive query `["GI₆", "DIŠ"], ["U", "BA", "MA"]]` looks for matching in lines which are next to each other

**Comment by ejimsan** [comment]
[Use display mode in Search](https://trello.com/c/nkFK6XRn/405-use-display-mode-in-search)

File: ebl/fragmentarium/domain/fragment.py [hunk 2]
```diff
@@ -116,11 +115,11 @@ def update_lemmatization(self, lemmatization: Lemmatization) -> "Fragment":
         text = self.text.update_lemmatization(lemmatization)
         return attr.evolve(self, text=text)
 
-    def get_matching_lines(self, query: TransliterationQuery) -> Lines:
+    def get_matching_lines(self, query: TransliterationQuery):
```
Comment (fsimonjetz) [inline ebl/fragmentarium/domain/fragment.py:118]: Maybe add return type `Text`?

File: ebl/tests/fragmentarium/test_dtos.py [hunk 3]
```diff
@@ -91,16 +92,16 @@ def test_create_response_dto(user):
 
 def test_create_fragment_info_dto():
     lemmatized_fragment = LemmatizedFragmentFactory.build()
-    line = "1. kur"
-    info = FragmentInfo.of(lemmatized_fragment, ((line,),))
+    line = parse_atf_lark("1. kur")
```
Comment (fsimonjetz) [inline ebl/tests/fragmentarium/test_dtos.py:95]: I'd suggest renaming `line` to `text`.

**Pull request merged**